frontend/src/components/EmployeeCards/EmployeeCards.tsx

```diff
@@ -128,7 +128,7 @@ const EmployeeCard = ({
         firstName={firstName}
         lastName={lastName}
         netId={netId}
-        photoLink={photoLink}
+        photoLink={photoLink ? photoLink + '?v=' + new Date().getTime() : ''}
```

Comment:
tiny thing but I think using string interpolation could make the string manipulation clearer